### ISSUE TITLE
Stratified sampling

### ISSUE THREAD

**Issue by Doidel** [i4Ds/SDOBenchmark#4]
Currently, the test set distribution differs from the training set distribution.
I'm not sure yet whether this is really an issue.
